XRTK-Core/Packages/com.xrtk.core/Definitions/InputSystem/MixedRealityInputAction.cs

```diff
@@ -30,32 +48,61 @@ public MixedRealityInputAction(uint id, string description, AxisType axisConstra
         /// <summary>
         /// Default input action that doesn't represent any defined action.
         /// </summary>
-        public static MixedRealityInputAction None { get; } = new MixedRealityInputAction(0, "None");
+        public static MixedRealityInputAction None { get; } = new MixedRealityInputAction(default, 0, "None");
```

Comment:
Could this be `readonly` so it be can be compile time constant?

Reply:
That's a c#8 language feature

Reply:
Sorry, I meant without the getter